tests/testthat/test-OTHER-create-track-hub.R

```diff
@@ -0,0 +1,154 @@
+library(testthat)
+library(PeakSegPipeline)
+library(data.table)
+context("create track hub")
+
+test.data.dir <- file.path(Sys.getenv("HOME"), "PeakSegPipeline-test")
+non.integer.dir <- file.path(test.data.dir, "non-integer")
+demo.dir <- file.path(test.data.dir, "input")
+hub.txt <- file.path(demo.dir, "hub.txt")
+trackDb.txt <- file.path(demo.dir, "trackDb.txt")
+
+download.to <- function
+(u, f, writeFun=if(grepl("bigWig", f))writeBin else writeLines){
+  if(!file.exists(f)){
+    require(httr)
+    f.dir <- dirname(f)
+    dir.create(f.dir, showWarnings=FALSE, recursive=TRUE)
+    request <- GET(u)
+    stop_for_status(request)
+    writeFun(content(request), f)
+  }
+}
+
+### Download bigWig files from github.
+bigWig.part.vec <- c(
+  "kidney/MS002201",
+  "bcell/MS010302"
+)
+
+label.txt <- "
```

Comment:
do we need labels to create the track hub? I think it should work without labels, and that would simplify your test.
smaller test is better (easier to understand / modify)

Reply:
As I was using the pipeline to generate `joint_peaks` file and it require labels in Step 0 that's the only reason I used labels

Reply:
it should work without joint_peaks.bigWig files, and that would be a simpler test